makenuget.cmd

```diff
@@ -0,0 +1,4 @@
+del "C:\NuGet\Remotion.BuildScript.2.0.0-alpha099.nupkg"
```

Comment:
This should be removed when you're done.

Reply:
Before we're done, there is bit left here.